HW10/task2.py

```diff
@@ -0,0 +1,53 @@
+# Написати кастомний Exception клас, MyCustomException, який має повідомляти "Custom exception is occurred".
+print("Hello! I am a simple phone book. My commands:\nstats, add, delete, list, show")
+
+class MyCustomException(Exception):
+    pass
+
+    def get_error_code(self):
+        return self.error_code
+
+phone_book = { "Adam" : 56474,
+               "Andreo" : 45453,
+               "Melisa" : 78475,
+}
+while True:
+    command = input("Enter command: ").lower()
+
+    if command == "stats":
+        print(f"I have {len(phone_book)} numbers")
+
+    if command == "add":
+        name = input("Name: ")
+        if name in phone_book:
+            print(f"You can't add {name}, it already exists")
+            continue
+        try:
+            number = input("Number: ")
+            if number.isdigit() == False:
+                raise MyCustomException
```

Comment:
nit: better to call an exception with a message 
```suggestion
                raise MyCustomException("Custom exception is occurred")
```